src/sentry/options/defaults.py

```diff
@@ -270,13 +270,6 @@
     flags=FLAG_ALLOW_EMPTY | FLAG_PRIORITIZE_DISK | FLAG_AUTOMATOR_MODIFIABLE,
 )
 
-register(
```

Comment:
Should you remove the option from sentry-options-automator first? https://github.com/getsentry/sentry-options-automator/pull/210

Reply:
err I'll ask about that

Reply:
Made PR here: https://github.com/getsentry/sentry-options-automator/pull/418